core/src/main/java/org/apache/brooklyn/core/mgmt/rebind/RebindIteration.java

```diff
@@ -1076,7 +1151,7 @@ protected Policy newPolicy(PolicyMemento memento) {
                 policy = invokeConstructor(null, policyClazz, new Object[] {flags});
             }
             
-            setCatalogItemId(policy, transformedCatalogItemId);
+            setCatalogItemIds(policy, memento.getCatalogItemId(), memento.getCatalogItemIdSearchPath());
```

Comment:
i think this should be:

> setCatalogItemIds(policy, loaded.catalogItemId, loaded.searchPath);

in case there were any transformations that were applied (by `load(...)`)